### PR TITLE
Update 4.14 Release notes

### PR DESCRIPTION
**Pull request by andrijapanicsb**
Added new features (need to see how many new features...number)

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
pinging everyone to please review and ADD any new features which I have missed (I did go through the PR list on the 4.14 milestone, but might have skipped some accidentally)?

**Comment by andrijapanicsb** [comment]
requesting docbuild

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ What's New in |release|
 Version |release| is a |version| LTS release adding almost *XXX* new features, enhancements and fixes 
 since 4.13. 
 
-Apache CloudStack |release| is a |version| LTS release with nearly 200 new features, enhancements and fixes since 4.13, such as enhanced hypervisor support, performance increases and more user configurable controls.  Highlights include:
+Apache CloudStack |release| is a |version| LTS release with over 200 new features, enhancements and fixes since 4.13, such as enhanced hypervisor support, performance increases and more user configurable controls.  Highlights include:
```
Comment (DaanHoogland) [inline source/releasenotes/about.rst:23]: why?

**Comment by andrijapanicsb** [comment]
LGTM for now, merging.

**Pull request merged**